data/apply-ne_disputed_border_suppression.sql

```diff
@@ -0,0 +1,512 @@
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+)),
+
+ru_ne_50m_admin_0_boundary_lines_land as (
+update ne_50m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_110m_admin_0_boundary_lines_land as (
+update ne_110m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_10m_admin_0_boundary_lines_disputed_areas as (
+update ne_10m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_50m_admin_0_boundary_lines_disputed_areas as (
+update ne_50m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+-- Add Russian disputed view to Nagorno-Karabakh
+ab_do_so_tr_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_ru = ''
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
+)),
+
+ab_do_so_tr_ne_50m_admin_0_boundary_lines_land as (
+update ne_50m_admin_0_boundary_lines_land l set
+            fclass_ru = ''
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
+
+    )),
+
+ab_do_so_tr_ne_110m_admin_0_boundary_lines_land as (
+update ne_110m_admin_0_boundary_lines_land l set
+            fclass_ru = ''
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
+
+    )),
+
+ab_do_so_tr_ne_10m_admin_0_boundary_lines_disputed_areas as (
+update ne_10m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = ''
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
+
+    )),
+
+ab_do_so_tr_ne_50m_admin_0_boundary_lines_disputed_areas as (
+update ne_50m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = ''
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
+
+    )),
+
+
+-- Add Turkish view to Northern Cyprus
+tr_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_tr = 'country'
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
```

Comment:
Is this for Northern Cyprus or nagorno_karabakh?

Reply:
Yeah good call. This isn't really ready for review yet. I'm just testing out the syntax